Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,588 @@
+    """
+    Create copy and color json values according to their values.
+    :param pred_json: json to color
+    :return: json
+    """
+    pred_json_colored = copy.deepcopy(pred_json)
+    pred_json_colored[MODEL_KEY_SEO] = MAPPING_VERDICT_TO_DISPLAY_VERDICT[MODEL_KEY_SEO][pred_json[MODEL_KEY_SEO]]
+    pred_json_colored[MODEL_KEY_LOGO_FOUND] = MAPPING_VERDICT_TO_DISPLAY_VERDICT[MODEL_KEY_LOGO_FOUND][
+        pred_json[MODEL_KEY_LOGO_FOUND]]
+    pred_json_colored[MODEL_KEY_LOGIN_FORM] = MAPPING_VERDICT_TO_DISPLAY_VERDICT[MODEL_KEY_LOGIN_FORM][
+        pred_json[MODEL_KEY_LOGIN_FORM]]
+    pred_json_colored[DOMAIN_AGE_KEY] = MAPPING_VERDICT_TO_DISPLAY_VERDICT[DOMAIN_AGE_KEY][pred_json[DOMAIN_AGE_KEY]]
+    return pred_json_colored
+
+
+def create_X_pred(output_rasterize: Dict, url: str) -> pd.DataFrame:
+    """
+    Create dataframe to predict from the rasterize output
+    :param output_rasterize: Dict from the output of rasterize command
+    :param url: url to examine
+    :return: pd.DataFrame
+    """
+    website64 = output_rasterize.get('image_b64', None)
+    html = output_rasterize.get('html', None)
+    X_pred = pd.DataFrame(columns=['name', 'image', 'html'])
+    X_pred.loc[0] = [url, website64, html]
+    return X_pred
+
+
+def prepend_protocol(url: str, protocol: str, www: bool = True) -> str:
+    """
+    Append a protocol name (usually http or https) and www to a url
+    :param url: url
+    :param protocol: protocol we want to add (usually http or https)
+    :return: str
+    """
+    p = urllib.parse.urlparse(url, protocol)  # type: ignore
+    netloc = p.netloc or p.path
+    path = p.path if p.netloc else ''
+    if not netloc.startswith('www.') and www:
+        netloc = 'www.' + netloc
+    p = urllib.parse.ParseResult(protocol, netloc, path, *p[3:])  # type: ignore
+    return p.geturl()
+
+
+def is_valid_url(url: str) -> Tuple[bool, str]:
+    """
+    Check is an url is valid by requesting it using different protocol
+    :param url: url
+    :return: bool
+    """
+    try:
+        response = requests.get(url, verify=False)  # nosec  guardrails-disable-line
+    except requests.exceptions.RequestException:
+        prepend_url = prepend_protocol(url, 'http', True)
+        try:
+            response = requests.get(prepend_url, verify=False)  # nosec guardrails-disable-line
+        except requests.exceptions.RequestException:
+            prepend_url = prepend_protocol(url, 'https', True)
+            try:
+                response = requests.get(prepend_url, verify=False)  # nosec guardrails-disable-line
+            except requests.exceptions.RequestException:
+                prepend_url = prepend_protocol(url, 'http', False)
+                try:
+                    response = requests.get(prepend_url, verify=False)  # nosec guardrails-disable-line
+                except requests.exceptions.RequestException:
+                    prepend_url = prepend_protocol(url, 'https', False)
```

Comment:
Can this get into a loop? 
+ if I try to use the script with a url which doesn't exist anymore, I get an error from the script without an explanation 
for instance, 
`!DBotPredictURLPhishing urls="www.wala.co.il"`